### PR TITLE
#1050 #1068 various default options tweaks

### PR DESCRIPTION
**Pull request by alex-courtis**
- remove defaults from bug template
- script to update defaults in readme, help
- move the following to DEFAULT_OPTS
  - view
  - trash
  - window picker
- add empty mapping defaults; this bugged me when I started, as I didn't know _where_ to put my mappings

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -63,108 +63,36 @@ body:
       value: |
         vim.cmd [[set runtimepath=$VIMRUNTIME]]
         vim.cmd [[set packpath=/tmp/nvt-min/site]]
-        local package_root = '/tmp/nvt-min/site/pack'
```
Comment (alex-courtis) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]: stylua

File: README.md [hunk 2]
```diff
@@ -108,68 +108,74 @@ highlight NvimTreeFolderIcon guibg=blue
 ```lua
 -- init.lua
 
--- following options are the default
+-- empty setup using defaults: add your own options
```
Comment (alex-courtis) [inline README.md:111]: This would have been very useful for me when starting out with nvim-tree.

**Review by kyazdani42 — APPROVED** [review]
just rebase it :)

**Comment by alex-courtis** [comment]
> just rebase it :)

ready

**Comment by alex-courtis** [comment]
Please merge this one first @kyazdani42 so that we don't get any further conflicts ;)

**Review by kyazdani42 — APPROVED** [review]

**Pull request merged**